packages/protocol/src/protocol/MessageExchange.ts

```diff
@@ -88,6 +88,9 @@ export type ExchangeSendOptions = {
 
     /** Additional context information for logging to be included at the beginning of the Message log. */
     logContext?: ExchangeLogContext;
+
+    /** Allows to override the protocol ID of the message, mainly used for standalone acks. */
+    protocolId?: number;
```

Comment:
It's weird to me that the message exchange is protocol independent except for this one feature.  Would it be possible to create a `SecureProtocolMessageExchange` that adds standalone ack support?  Or to do it with composition, have some sort of plugin component with hooks in all of the places where it's wired in?

I know standalone ack is pretty well intertwined with the code here so maybe better as a future TODO.